docroot/sites/development.services.yml

```diff
@@ -4,6 +4,13 @@
 # 'example.settings.local.php' file, which sits next to this file.
 parameters:
   http.response.debug_cacheability_headers: true
+  twig.config:
```

Comment:
@weekbeforenext I noticed that everytime I run `ahoy comi` these changes get overridden. I think this is generated by something else.

Reply:
I added a message to Teams to see how to preserve these changes for local dev. It looks like they are being replaced by core scaffold files in the composer.lock file.

Reply:
I think this is fixed in composer now.